toolkit/featureforms/src/androidTest/java/com/arcgismaps/toolkit/featureforms/DateTimeFieldTests.kt

```diff
@@ -0,0 +1,260 @@
+     * when it is not focused
+     * the text values are as expected
+     * https://devtopia.esri.com/runtime/common-toolkit/blob/main/designs/Forms/FormsTestDesign.md#test-case-21-unfocused-and-focused-state-no-value-date-required
+     * https://devtopia.esri.com/runtime/common-toolkit/blob/main/designs/Forms/FormsTestDesign.md#test-case-22-focused-and-unfocused-state-with-value-populated
+     * (the test is a hybrid of 2 tests because the field has a value and the table cannot be edited)
+     */
+    @Test
+    fun testRequiredUnfocusedValue() {
+        val formElement = getFormElementWithLabel("Required Date")
+        // find the field with the the label
+        val col = composeTestRule.onNodeWithContentDescription("lazy column")
+        col.performScrollToIndex(8)
+        val dateTimeField = composeTestRule.onNodeWithText("${formElement.label} *")
+
+        val textMatcher = hasText("No value")
+        assert(textMatcher.matches(dateTimeField.fetchSemanticsNode())) {
+            "expected text: No value."
+        }
+        dateTimeField.assertIsDisplayed()
+        val helper = dateTimeField.onChildWithContentDescription("supporting text")
+        val helperMatcher = hasText("Date Entry is Required")
+        assert(helperMatcher.matches(helper.fetchSemanticsNode())) {
+            "expected helper text: Date Entry is Required"
+            
+        }
+        val iconMatcher = hasContentDescription("field icon")
+        assert(iconMatcher.matches(dateTimeField.fetchSemanticsNode()))
+    }
+    
+    /**
+     * Given a required datetime field with a value
+     * when it is not focused
+     * the text values are as expected
+     * https://devtopia.esri.com/runtime/common-toolkit/blob/main/designs/Forms/FormsTestDesign.md#test-case-21-unfocused-and-focused-state-no-value-date-required
+     * https://devtopia.esri.com/runtime/common-toolkit/blob/main/designs/Forms/FormsTestDesign.md#test-case-22-focused-and-unfocused-state-with-value-populated
+     * (the test is a hybrid of 2 tests because the field has a value and the table cannot be edited)
+     */
+    @Test
+    fun testRequiredFocusedValue() {
+        val formElement = getFormElementWithLabel("Required Date")
+        // find the field with the the label
+        val col = composeTestRule.onNodeWithContentDescription("lazy column")
+        col.performScrollToIndex(8)
+        val dateTimeField = composeTestRule.onNodeWithText("${formElement.label} *")
+        val iconMatcher = hasContentDescription("field icon")
+        assert(iconMatcher.matches(dateTimeField.fetchSemanticsNode()))
+        dateTimeField.assertHasClickAction()
+        dateTimeField.performClick()
+    
+        val dialogSurface =
+            composeTestRule.onNodeWithContentDescription("DateTimePickerDialogSurface", useUnmergedTree = true)
+        dialogSurface.assertIsDisplayed()
+        saveScreenshot("xx",  dialogSurface.captureToImage().asAndroidBitmap())
```

Comment:
```suggestion
        saveScreenshot("testRequiredFocusedValue",  dialogSurface.captureToImage().asAndroidBitmap())
```
It might be easy look up the test results if saved with the test name.